sdk/js/src/utils/array.ts

```diff
@@ -96,7 +98,7 @@ export const tryUint8ArrayToNative = (
     // wormhole-chain addresses are always 20 bytes.
     return humanAddress("wormhole", a.slice(-20));
   } else if (chainId === CHAIN_ID_NEAR) {
-    throw Error("uint8ArrayToNative: Near not supported yet.");
+    throw Error("uint8ArrayToNative: Use tryHexToNativeStringNear instead.");
```

Comment:
Can't you just call that function here and return it?

Reply:
We made `tryHexToNativeStringNear` a separate `async` function, because making `tryUint8ArrayToNative` `async` would be a burden for integrators to change